helpers.lua

```diff
@@ -244,8 +244,17 @@ function helpers.sysctl_async(path_table, parse)
 
     spawn.with_line_callback("sysctl " .. path, {
         stdout = function(line)
+	    local separators = {
+	       openbsd = "=",
+	       linux = " = ",
+	       freebsd = ": "
+	    }
+
+	    local pattern = "(.+)%s(.+)"
+	    pattern = pattern:format(separators[helpers.getos()])
+
             if not string.find(line, "sysctl: unknown oid") then
```

Comment:
Since it is not shown in your screenshot, is `sysctl`'s error on OpenBSD has the same format?

Reply:
It doesn't, but that scenario is now taken care of. I could use help testing it on Linux, and FreeBSD though.